scripts/compress_assets.sh

```diff
@@ -7,9 +7,9 @@ set -euo pipefail
 cd ui/react-app
 cp embed.go.tmpl embed.go
 
-GZIP_OPTS="-fk"
+GZIP_OPTS="-fkn"
 # gzip option '-k' may not always exist in the latest gzip available on different distros.
-if ! gzip -k -h &>/dev/null; then GZIP_OPTS="-f"; fi
+if ! gzip -k -h &>/dev/null; then GZIP_OPTS="-fn"; fi
```

Comment:
My only worry is that browsers might use these headers to know whether they should cache the bundle, but a quick Google search yielded no results.

I'll let @Nexucis and @juliusv advise on whether this is used but I'll merge for now because overall it's a nest positive for upstream consumers.